src/main/groovy/org/shipkit/internal/notes/header/HeaderProvider.java

```diff
@@ -0,0 +1,11 @@
+package org.shipkit.internal.notes.header;
+
+public class HeaderProvider {
+
+    public static final String HEADER_START = "<sup><sup>";
```

Comment:
Yes exactly. They are invisible. On markdown in GitHub you can add even comment which is not visible